web/frontend/src/components/collapse.svelte

```diff
@@ -0,0 +1,58 @@
+<!--
+  Rendering embedded maps inside of a shadow dom is currently broken for MapLibreGL.
+  This Collapse component is taken from PRIME but removes all web component behavior.
+  Once we've completed the svelte migration it can be removed and replaced with the PRIME component.
+-->
```

Comment:
That's interesting.

Reply:
Yeah, this one is hard to debug. I'm actually not sure where the bug is. It could be in MapLibreGL, svelte's custom element rendering pipeline, etc 🤷